core/src/main/java/org/jdbi/v3/core/argument/EnumArgumentFactory.java

```diff
@@ -49,7 +51,18 @@
     }
 
     private static <E extends Enum<E>> Optional<Argument> byName(E value, ConfigRegistry config) {
-        return makeArgument(Types.VARCHAR, String.class, value, E::name, config);
+        return makeArgument(Types.VARCHAR, String.class, value, e -> {
```

Comment:
this may be more suited to a private method, being such a big lambda in an already long line of code

Reply:
I moved the lambda expression out of the method call.